client/src/pages/Profile/Profile.tsx

```diff
@@ -0,0 +1,76 @@
+import { Grid, Paper, Typography, Button } from '@material-ui/core/';
+import { useState } from 'react';
+import useStyles from './useStyles';
+import LocationOnIcon from '@mui/icons-material/LocationOn';
+import TextField from '@mui/material/TextField';
+import AdapterDateFns from '@mui/lab/AdapterDateFns';
+import LocalizationProvider from '@mui/lab/LocalizationProvider';
+import DateTimePicker from '@mui/lab/DateTimePicker';
+export default function Profile(): JSX.Element {
+  const classes = useStyles();
+  const [dropInValue, setDropInValue] = useState<Date | null>(new Date());
+  const [dropOffValue, setDropOffValue] = useState<Date | null>(new Date());
+  return (
+    <Grid container>
+      <Paper className={classes.profileContainer}>
+        <img className={classes.coverImage} src="/pics/cover-sample.jpg" alt="Cover Photo" />
+        <Grid container className={classes.basicInfoContainer} direction="column" alignItems="center">
+          <img className={classes.profilePic} src="/pics/profilepic-sample.jpg" alt="Profile Pic" />
+          <Typography variant="h4">Norma Byers</Typography>
```

Comment:
Let's extract these hardcoded profile details into a `mockProfile` and use that to fill in this page. You can extract the `img` tag sources as well.

Reply:
mockProfile created and now all info comes from mockProfile. :)